app/models/concerns/fog_extensions.rb

```diff
@@ -15,12 +15,10 @@ module FogExtensions
 
 if Foreman::Model::GCE.available?
   require 'fog/google'
-  require 'fog/google/models/compute/image'
+  require 'fog/compute/google/models/image'
   Fog::Compute::Google::Image.send(:include, FogExtensions::Google::Image)
-  require 'fog/google/models/compute/server'
+  require 'fog/compute/google/models/server'
   Fog::Compute::Google::Server.send(:include, FogExtensions::Google::Server)
-  require 'fog/google/models/compute/flavor'
-  Fog::Compute::Google::Flavor.send(:include, FogExtensions::Google::Flavor)
```

Comment:
If this is no longer used should `app/models/concerns/fog_extensions/aws/flavor.rb` be also removed?

Reply:
As `flavor` model exists in `fog-aws`, we don't need to remove this model.

+1,  If I understood correctly then you would like to mention `app/models/concerns/fog_extensions/google/flavor.rb` here, right?
I will remove the file app/models/concerns/fog_extensions/google/flavor.rb.